tex/texlive-common/Portfile

```diff
@@ -27,7 +27,7 @@ checksums           rmd160  6fb366c4bddd6b98840d1e852af123ad25ffeafa \
 
 livecheck.type  regex
 livecheck.url   ${master_sites}
-livecheck.regex ${name}-(\[\\d-\]+)\\.tar
+livecheck.regex ${name}-(\[\\d-\]+(?:\\.\\d+)?)\\.tar
```

Comment:
Maybe you should make this `(?:\\.\\d+)*` so the group can occur an arbitrary number of times, just in case texlive ever choose to introduce patch-level version numbers?